### PR TITLE
fix(package.json): fix export map types

### PR DESCRIPTION
**Pull request by tazyong**
Fix import error in typescript
image

### CODE REVIEW

**Review by bachirelkhoury — APPROVED** [review]
This resolved the typing error

**Comment by trusktr** [comment]
This will be handled soon as part of

- https://github.com/tweenjs/tween.js/issues/655.

Besides it won't make sense to point both ESM and CommonJS files to the same types.

So in #655 we'll work on this next first thing, simplify the output, moving onto only ES Modules.

File: package.json [hunk 1]
```diff
@@ -8,8 +8,14 @@
 	"module": "dist/tween.esm.js",
 	"exports": {
 		".": {
-			"import": "./dist/tween.esm.js",
```
Comment (july-12) [inline package.json:11]: could use more easy way by just one line inserted like this ?
image
it seems work for me!

**Comment by DJ-caddev** [comment]
Any update on this issue?

**Comment by MasatoMakino** [comment]
Here is a minimal tsconfig.json to reproduce this problem.

```json
{
  "compilerOptions": {
    "module": "ES2015",
    "noImplicitAny": true,
    "moduleResolution": "bundler" // or "node16"
  }
}
```
`noImplicitAny` is [part of `strict`](https://www.typescriptlang.org/tsconfig#strict).

I hope this record helps someone.

**Comment by trusktr** [comment]
Thanks @MasatoMakino. Will circle to this soon, this week.

**Comment by Methuselah96** [comment]
Hey @trusktr this is a pretty rough for TypeScript users, since TypeScript doesn't think there are any types for modern `moduleResolution` settings. I had to [copy-and-paste the types into `@types/three`](https://github.com/three-types/three-ts-types/pull/593) to work around this.

Looking forward to this getting resolved. :)

**Comment by trusktr** [comment]
The types have been recently added from the other PR (the one-liner as @july-12 mentioned above)!

**Comment by trusktr** [comment]
Thank you for the contribution!

**Pull request merged**